zerver/migrations/0426_realmuserdefault_do_not_mark_messages_as_read_and_more.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.1.6 on 2023-02-06 02:01
```

Comment:
The name for migration file could be better here. Something like `0426_add_do_not_mark_messages_as_read_setting`